Jenkinsfile

```
@@ -123,7 +123,7 @@ def qb_inst_rpms = ""
 el7_component_repos = ""
 def functional_rpms  = "--exclude openmpi openmpi3 hwloc ndctl " +
                        "ior-hpc-cart-4-daos-0 mpich-autoload-cart-4-daos-0 " +
-                       "romio-tests-cart-4-daos-0 hdf5-tests-cart-4-daos-0 " +
+                       "romio-tests-cart-4-daos-0 hdf5-mpich-tests " +
```

FWIW, this should use the virtual requirement that targets the daos (and cart, but this actually can be eliminated, which we probably should do in 1.12) major versions:
```suggestion
                       "romio-tests-cart-4-daos-0 hdf5-mpich-tests-cart-4-daos-0 " +
```